README.md

```diff
@@ -1,141 +1,146 @@
-The monetary policy of ANT is intended to be mostly algorithmic, automatically responding to changes to price and participation rates to promote the healthy growth of the network. In order to respond to these variables the algorithm must be parameterized with a set of target values for price, stability, and various participation rates.
+In order for the court to operate fees must be captured from users to compensate jurors for their effort, risk, and opportunity cost of capital. The market for dispute resolution services exhibits some dynamics of a predator-prey relationship. Disputes will be more rare if there is a common expectation that the dispute resolution process is reliable and fair. The result is that as disputes become more rare, or are settled in early dispute rounds, the incentive to participate as a juror decreases. As these incentives decrease there will be less stake in the court and the court may be perceived as less reliable, resulting in more disputes and more appeals leading to overall less consistent and efficient operation. To resolve this issue, fees are captured not just when disputes occur, but also on a recurring basis from users even when no disputes occur.
 
-Changes to these parameters can be constrained so that adjustments must be made in small increments and significant changes would need to be done gradually over a long period of time.
-### 4.3.1 Price Target
-The price target determines when the supply of ANT is expanded or contracted. For simplicity of implementation this could be set to a common unit of account like the USD, but ideally would be set to an inflation adjusted basket of goods similar to the CPI. Targeting a specific currency requires little intervention, whereas, targeting a basket of goods or basket of currencies requires the network to manage the elements and weights of the basket.
-### 4.3.2 Stability Target
-The stability target determines how conservative the automated reserve policy is. With a high stability target the reserve will hold more collateral, however, as a result holders of the currency will experience less value appreciation through currency splits. It is likely that the optimal stability target will change over time as the network and currency grows and becomes inherently more stable.
-### 4.3.3 Participation Targets
-Participation targets are used to incentivize important contributions to the network including maintaining a desired ratio between collateralized and liquid ANT, rewarding price feed providers, rewarding dispute reviewers, and funding the treasury. In general participation targets look at the ratio of ANT collateralized for a specific purpose relative to the total supply.
+- **Agreement Tax**: A tax is imposed on organizations which choose to use the court as an arbitration provider. Taxes must be paid on a recurring basis for the agreement to remain valid. These fees are split between the Aragon Network's treasury and among all actively staked jurors.
```

Comment:
Why call it a "tax" instead of a fee? It's even called a fee later in the description:

> These fees are split between the Aragon Network's treasury and among all actively staked jurors.

It seems like a subscription fee.